docs/src/references/c8y-firmware-management.md

```diff
@@ -0,0 +1,214 @@
+# Device Firmware Management using Cumulocity
+
+Thin-edge provides an operation plugin to
+[manage device firmware using Cumulocity](https://cumulocity.com/guides/users-guide/device-management/#firmware-repo).
+Firmware management is currently supported only for child devices and not for the main tedge device.
+
+## Installation
+
+The plugin will be installed at `/usr/bin/c8y-firmware-plugin` by the debian package.
+The systemd service definition files for the plugin are also installed at `/lib/systemd/system/c8y-firmware-plugin.service`.
```

Comment:
Do we need mention the exact paths here? Or would it just be enough to say:

```suggestion
The plugin includes a single executable called `c8y-firmware-plugin` and a systemd service definition with the same name.
```

Reply:
If it was user facing how-to guide, I'd have definitely gone with your version. But, since this is a plugin specification which the implementor should have used to create the debian package definition as well (if this doc existed before the dev implemented it  ;), of course), the details should be specific, right?

Reply:
Fair enough. You can leave it as is then :)